connector/solarwindsentityconnector/testdata/logsToLogs/relationship/expected-log-different-type-relationship.yaml

```
@@ -0,0 +1,88 @@
+resourceLogs:
+  - resource: {}
+    scopeLogs:
+      - logRecords:
+          - attributes:
+              - key: otel.entity.type
+                value:
+                  stringValue: Snowflake
+              - key: otel.entity.id
+                value:
+                  kvlistValue:
+                    values:
+                      - key: snowflake.id
+                        value:
+                          stringValue: snowflake-instance-id-first
```

Huge nit: 
```suggestion
                          stringValue: snowflake-instance-id
```

for all values, so we keep these yaml files clean as possible.

---

fixed